README.md

```diff
@@ -1,2 +1,25 @@
-# HW01
-Edit the readme
+---
+title: HW01 
```

Comment:
Nice way to include title, author, and date! I will try this in my code next time.

Reply:
I also think this is a really nice way to start the whole bio.